internal/web/routes/routes.go

```diff
@@ -0,0 +1,18 @@
+package routes
+
+import (
+	"context"
+
+	"github.com/google/go-github/v68/github"
+	"github.com/labstack/echo/v4"
+	"github.com/openai/openai-go"
+)
+
```

Comment:
The route patterns for the GET requests should validate the parameters involved (like `repo`, `pull_number`, and `owner`) to prevent issues stemming from invalid data. Consider implementing middleware or validation logic to ensure these values adhere to expected formats before proceeding.